pkgs/applications/science/logic/coq/default.nix

```diff
@@ -129,14 +129,15 @@ self = stdenv.mkDerivation {
     ++ optionals buildIde
       (if versionAtLeast "8.10"
        then [ ocamlPackages.lablgtk3-sourceview3 glib gnome.adwaita-icon-theme wrapGAppsHook ]
-       else [ ocamlPackages.lablgtk ]);
+       else [ ocamlPackages.lablgtk ])
+    ++ optional (versionAtLeast "8.14") [ ocamlPackages.dune_2 ]
```

Comment:
Should have been `optional (versionAtLeast "8.14") ocamlPackages.dune_2`

Reply:
@Zimmi48

Reply:
I've opened #128747.